scss/components/side-nav.scss

```diff
@@ -100,12 +101,11 @@ $block: #{$fd-namespace}-side-nav;
     }
 
     &__link {
-
         //STATES *******************************************
         &.has-child {
             background-image: url(#{$fd-down-arrow-icon});
             background-repeat: no-repeat;
-            background-position: calc(100% - #{$fd-forms-select-width--background-image}) center;
+            background-position: left calc(100% - 14px) top 15px;
```

Comment:
I think this can be simplified to avoid using the `calc` function.  If believe the following line achieves the same results:
```scss
background-position: right 14px top 15px;
```

Reply:
yeah, it will give the same position. thanks for the comments.

Reply:
`calc()` is a perf sink. It should only ever be used specific use cases. (see https://codepen.io/nicolasjengler/post/don-t-misuse-the-calc-css-function)

we should actually take a pass where it can refactored out.